client/pages/resetPassword.js

```diff
@@ -0,0 +1,81 @@
+import {
```

Comment:
In the reset page we only accept 
- New password 
- Confirm Password

No email